Parse-Dashboard/parse-dashboard-config.json

```diff
@@ -2,13 +2,15 @@
   "apps": [
     {
       "serverURL": "http://localhost:1338/parse",
-      "appId": "hello",
```

Comment:
Can you please revert? I already deleted the index.html file.

Reply:
I've reverted these changes.